singer-io/tap-github
# TDL 16573 sync teams at organization level

# Description of change
- Sync `teams`, `team_members`, and `team_memberships` single time only for each organization.
- Sync all other streams for every repo.
# Manual QA steps
 - Provide  multiple repos with common organization and verify that the above 3 streams sync only 1 time.
 - Provide  multiple repos with common organization and verify that all streams except the above 3 streams sync for each repo.
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
